install_config/aggregate_logging_sizing.adoc

```diff
@@ -49,10 +49,10 @@ Refer to
 xref:../install_config/aggregate_logging.adoc#aggregate-logging-ansible-variables[Ansible
 Variables] for a full list of parameters.
 
-Kibana requires a hostname that can be resolved from wherever the browser will be used to access it.  
-For example, you might need to add a DNS alias for Kibana to your corporate name service in order to access Kibana 
-from the web browser running on your laptop.  Logging deployment creates a Route to Kibana on one of your "infra" nodes 
-or wherever the OpenShift router is running.  The Kibana hostname alias should point to this machine.  
+Kibana requires a hostname that can be resolved from wherever the browser will be used to access it.
```

Comment:
s/hostname/host name per IBM Style Guide

Reply:
lol that's weird--I didn't write/edit this section. Fixed it though :) just curious as to why it came up here